src/adapters/scene_direction.js

```diff
@@ -4,22 +4,46 @@ import { normalizeToFeatureCollection } from 'src/libs/geojson';
 import { map } from '../../config/constants.yml';
 import LatLonPoi from '../adapters/poi/latlon_poi';
 import { prepareRouteColor, getRouteStyle, setActiveRouteStyle } from './route_styles';
-import { getAllSteps, getAllStops, originDestinationCoords } from 'src/libs/route_utils';
+import {
+  getAllSteps,
+  getAllStops,
+  originDestinationCoords,
+  formatDuration,
+  formatDistance,
+  getVehicleIcon,
+} from 'src/libs/route_utils';
 import Error from '../adapters/error';
 import nconf from '@qwant/nconf-getter';
 import { fire, listen } from 'src/libs/customEvents';
+import { getLabelPositions } from 'src/libs/route_labeler';
 
 const createMarker = (lngLat, className = '', options = {}) => {
   const element = document.createElement('div');
   element.className = className;
   return new Marker({ ...options, element }).setLngLat(lngLat);
 };
 
+const createRouteLabel = (route, vehicle, { lngLat, anchor }) => {
```

Comment:
this function would best fit in `route_labeler.js`, what do you think ?

Reply:
I don't know, I wrote this function of the model of `createMarker` above and the idea was to separate the work of the lib on raw stuffs from the MapBox dependency and design-dependant markup.